### DISCUSSION TITLE
p-dropdown issue "version": "16.7.0"

### DISCUSSION THREAD

**Discussion by nidmiapp** [primefaces/community#729] [PrimeNG]
hello everyone, I don't know why p-dropdown always shows the first element of options. 
In ngOnInit I setValue any other element(for example 3rd element) and assume to see a dropdown with 3rd value in UI. but it won't happen.

**Comment by cetincakiroglu** [comment]
This issue has been fixed between versions 16.9-17.2. Could you please try it with the latest version of the PrimeNG (17.2.0)
